### ISSUE TITLE
Cropping interface is quite disturbing

### ISSUE THREAD

**Issue by liar666** [maoschanz/drawing#555]
Hi,

Thanks for the great tool. It's mostly a lot easier/faster to use than opening Gimp for small modification of images (== what I do most of the time).

My only critic would be on the crop tool, which UX is very disturbing : having to move the image/canvas to crop is very strange. Also this makes its usage very incoherent as for cropping in one direction (e.g. to the right) you move the canvas sand in the opposite direction  (e.g. to the left) you manipulate the anchors displayed on the corners

**Comment by maoschanz** [comment]
you don't have to move anything aside your mouse, the image moves as a mere side effect of you changing its origin

you move the pointer left when you want to move a side of the image to the left, that's not incoherent

**Comment by mishurov** [comment]
I agree with @liar666. Cropping is really disturbing. Doing Rectangle Selection, copying it into clipboard and doing New Image From Clipboard works more like traditional cropping.
